### DISCUSSION TITLE
Change docker internal port from 8080 to something else

### DISCUSSION THREAD

**Discussion by tugdualenligne** [TandoorRecipes/recipes#1118] [Q&A] [answered]
Hi there, many thnaks for appears a niece piece of work (haven't tried it yet). 
I'm struggling to have it work because I reckon I have other container with port 8080 exposed to my Traefik reverse proxy specific network (and therefore I can't map ports, have to use the container's internal one.
Is there a way to change the 8080 port to something less usual please? (this is also Traefic API's default port and the common port for proxies in general)

**Comment by vabene1111** [comment, marked as answer]
there should be no need to change the port if you are using traefik. I have probabaly 10 different services all running on port 80 in my traefik network, the whole point of traefik is that this is possible and you dont need to change around so many ports but can all reach them trough the single port of entry.

Also you could just use docker to map 8080 container side to whatever you want host side.

**Reply by tugdualenligne** [reply]
You're right, I managed to make this work!
